../utils.py:70: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  gridspec_kw={"height_ratios": [1, 1, 3, (n_components * 0.1)]},
/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-1/06_ica_global_signal.py:213: RuntimeWarning: invalid value encountered in long_scalars
  np.sum(acc_corrs > 0.3) / acc_corrs.size
/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-1/06_ica_global_signal.py:217: RuntimeWarning: invalid value encountered in long_scalars
  np.sum(rej_corrs > 0.3) / rej_corrs.size
/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-1/06_ica_global_signal.py:227: RuntimeWarning: invalid value encountered in long_scalars
  np.sum(acc_corrs > 0.5) / acc_corrs.size
/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-1/06_ica_global_signal.py:231: RuntimeWarning: invalid value encountered in long_scalars
  np.sum(rej_corrs > 0.5) / rej_corrs.size
/home/data/nbc/misc-projects/Salo_PowerReplication/code/conda_env/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3420: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/home/data/nbc/misc-projects/Salo_PowerReplication/code/conda_env/lib/python3.7/site-packages/numpy/core/_methods.py:188: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-1/06_ica_global_signal.py:221: RuntimeWarning: invalid value encountered in long_scalars
  np.sum(ign_corrs > 0.3) / ign_corrs.size
/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-1/06_ica_global_signal.py:235: RuntimeWarning: invalid value encountered in long_scalars
  np.sum(ign_corrs > 0.5) / ign_corrs.size
